src/components/sandboxLink.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import usePlatform from "./hooks/usePlatform";
+
+const scenarios = [
+    'performance',
+    'releases',
+    'alerts',
+    'discover',
+    'dashboards',
+    'projects',
+    'oneDiscoverQuery',
+    'oneIssue',
+    'oneBreadcrumb',
+    'oneStackTrace',
+    'oneTransaction',
+    'oneWebVitals',
+    'oneTransactionSummary',
+    'oneRelease',
+  ] as const;
+
+type Props = {
+    children: React.ReactNode;
+    scenario?: typeof scenarios[number];
+    projectSlug?: string;
+    errorType?: string;
+    platform?: string;
```

Comment:
@Neo-Zhixing are we even using `platform`?

Reply:
Yes, it's used as in the `usePlatform` calls, which allows you to specify the platform manually. This seems to be the standard practice in the docs repo.